stablehlo/dialect/VhloAttrs.td

```diff
@@ -203,4 +166,55 @@ def VHLO_ConvolutionAttributes {
   );
 }
 
+//===----------------------------------------------------------------------===//
+// Forked Attributes
+//===----------------------------------------------------------------------===//
+
+def VHLO_ArrayAttr : VHLO_AttrDef<"ArrayV1"> {
+  let mnemonic = "array";
+  let parameters = (ins ArrayRefParameter<"mlir::Attribute">:$value);
+  let assemblyFormat = "`<` custom<AttributeArray>($value) `>`";
+}
+
+def VHLO_DenseIntOrFPElements : VHLO_AttrDef<"DenseIntOrFPElementsV1"> {
+  let mnemonic = "dense";
+  let parameters = (ins "::mlir::Type":$type, "ArrayRef<char>":$raw_data);
```

Comment:
Changed to:

```
def VHLO_DenseIntOrFPElementsData : TypeParameter<"::llvm::ArrayRef<char>", "Array of int"> {
  // Custom allocator to copy dense elements data into MLIR Context
  let allocator = "$_dst = $_allocator.copyInto($_self);";
}
def VHLO_DenseIntOrFPElementsAttrV1 : VHLO_AttrDef<"DenseIntOrFPElementsV1"> {
  let mnemonic = "dense";
  let parameters = (ins "::mlir::Type":$type, VHLO_DenseIntOrFPElementsData:$raw_data);
  let hasCustomAssemblyFormat = 1;
}
```

Curious what impact this will have on performance. Might not be too important since this is a serialization dialect.